generic/tests/unattended_install_reboot_driftfix.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from virttest import error_context
+
+from virttest.tests import unattended_install
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Unattended install test:
+    1) Starts a VM with an appropriated setup to start an unattended OS install.
+    2) Wait until the install reports to the install watcher its end.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    params["cpu_model_flags"] = ""
+    unattended_install.run(test, params, env)
```

Comment:
For this case, the vm was not expected to be shutdown after `Guest reported successful installation`. When installation as an individual case, if shutdown failed, the framework quit/kill vm forcibly in pose process before test case finish. 
Now, this case can pass because test does not finish, vm is still running.
Could you try adding `shutdown_cleanly =  no` ?

Reply:
@yanan-fu  I checked the unattended_install.py, becaues medium is cdrom, vm.shutdown is not executed.
1491     if params.get("shutdown_cleanly", "yes") == "yes":
1492         shutdown_cleanly_timeout = int(params.get("shutdown_cleanly_timeout",
1493                                                   120))
1494         logging.info("Wait for guest to shutdown cleanly")
1495         if params.get("medium", "cdrom") == "import":
1496             vm.shutdown()
1497         try:
1498             if utils_misc.wait_for(vm.is_dead, shutdown_cleanly_timeout, 1, 1):
1499                 logging.info("Guest managed to shutdown cleanly")
1500         except qemu_monitor.MonitorError as e:

Reply:
Yes, i see, but it go to `Wait for guest to shutdown cleanly`, this is not correct although it does not effect the final result.

Reply:
After adding shutdown_cleanly = no, the case passed.

Reply:
Could you add it to the cfg accordingly~